<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SongTableViewContainer</class>
 <widget class="QWidget" name="SongTableViewContainer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="SongTableView" name="tableView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>27</width>
         <height>27</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>27</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SongTableView</class>
   <extends>QTableView</extends>
   <header>SongTableView/songtableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
